CONTRIBUTE.md

```diff
@@ -0,0 +1,333 @@
+# JavaScript Interactive Examples Contributing Guide
+
+A guide for people wanting to contribute interactive JavaScript examples.
+
+## Naming Things
+
+It probably makes sense to name things in a consistent way throughout all examples.  Where possible the naming should ‘reflect’ the type of that object or variable.
+
+### Numerical Suffixes
+
+One way we could maintain a certain consistency is to use the type of the object with a numerical suffix.
+
+For example:
+
+```
+object1 = {};
+var string1 = “a string”;
+var array1 = ['a', 'b', 'c'];
+var array2 = [1, 2, 3];
+```
+
+Note it is suggested that even if there is only one instance of that object then it should be suffixed with a `1`.
+
+Note also although we could use the same name for multiple instances, it's suggested that we use a progressive numbering system to minimise confusion should the reader want to experiment with the example code.
+
+For example:
+
+```
+var array1 = ['a', 'b', 'c'];
+
+console.log(array1);
+// expected output: Array ["a", "b", "c"]
+
+var array2 = [1, 2, 3];
+
+console.log(array2);
+// expected output: Array [1, 2, 3]
+```
+
+An alternative to using numerical suffixes could be to use more descriptive suffixes, such as `arrayLetters` and `arrayNumbers` in the above examples.
+
+This may make more sense in the context of the following example:
+
+```
+var collatorDe = new Intl.Collator('de', { usage: 'search', sensitivity: 'base' });
+var collatorFr = new Intl.Collator('fr', { usage: 'search', sensitivity: 'base' });
+```
+
+### Indicating console.log output
+
+To indicate the ouput we expect, we place a comment `// expected output:` on the line below each `console.log`.
+
+
+### Representing Browser Differences
```

Comment:
Again I'm not sure if this is intended to be an H3 under "Naming Things"

Reply:
Similar to above, it's more wording than naming.